_posts/2019-07-03-reinforce-2019-recap.md

```diff
@@ -0,0 +1,104 @@
+
+![SDD334](/img/reinforce-2019-recap/day1-sdd334.jpg)
+
+## [SDD306 - Securing serverless and container services](https://www.youtube.com/watch?v=kmSdyN9qiXY)
+A session with a pretty good overview of the security features available in different layers of serverless and container services, focusing on compute services (Lambda, Fargate ECS, EKS). The talk finished with an example of integrating security testing steps into CodePipeline using Lambdas.
+
+![SDD306](/img/reinforce-2019-recap/day1-sdd306.jpg)
+
+## SDD323 - Automating remediation of noncompliant configurations
+A builders session focused on using AWS Config and creating a Lambda function for remediating configuration errors (e.g. a non-compliant S3 bucket policy). [Link to materials](https://sdd323-r-reinforce-2019.s3.amazonaws.com/automating-remediation-of-noncompliant-configurations.pdf)
+
+![SDD323](/img/reinforce-2019-recap/day1-sdd323.jpg)
+
+## SDD310 - DevSecOps: Integrating security into pipelines
+A pretty nice workshop with a long introduction to the topic, also with some interaction with attendees. After the long introduction, the workshop finished with a small lab for adding a Lambda function into an existing CodePipeline setup that checked if the built code contained any AWS credentials. [Link to materials](http://tinyurl.com/yx9yuhxg) with some nice slides on DevSecOps.
+
+![SDD310](/img/reinforce-2019-recap/day1-sdd310.jpg)
+
+After the first day, there were numerous unofficial after parties or happy hour events sponsored by companies at nearby restaurants. Those were again great for meeting new people and hearing how companies are using AWS.
+
+# Day 2
+
+## Jam Lounge
+In last years re:Invent, I stumbled across the jam lounge only on the final day. Basically the idea was that you could register to a website where you could complete different AWS related challenges during the event and compete for prizes. This time, I found the lounge early on the first day, so I wanted to see if I could compete for the top prizes.
+
+After the first day, I had already solved 4 of the 5 available daily challenges and had some issues launching the fifth challenge. Luckily with some help from the Jam lounge personnel early on the second day, I was able to get it working.
+
+I started to do the second day's challenges right after they were released and had some extra time to work on them during a workshop while waiting for Cloudformation stacks to finish creating. The challenges were surprisingly easy, so I managed to complete them all during the workshop and was the first one to finish all of them and won the first prize (Bose QC35 headphones)!
```

Comment:
`I managed to complete them all during the workshop and was the first one to finish all of them and won the first prize` a bit repetitive, could use restructuring.